tests/test_main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+from app.main import is_isogram
+
```

Comment:
Could you add tests with isograms?
For now, function like this will pass all tests:
```
def is_isogram(word: str):
    return len(word) == 0
```